src/Razor/src/Microsoft.AspNetCore.Razor.Language/AllowedChildTagDescriptor.cs

```diff
@@ -13,6 +13,8 @@ public abstract class AllowedChildTagDescriptor : IEquatable<AllowedChildTagDesc
 
         public string DisplayName { get; protected set; }
 
+        public bool CaseSensitive { get; protected set; }
```

Comment:
I assume this breaking change is ok.

Reply:
What's breaking about it?

Reply:
I meant to add that comment on the builder. Ignore